src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al

```diff
@@ -131,6 +131,18 @@ table 309 "No. Series Line"
             Caption = 'Temporary Sequence Number';
             DataClassification = SystemMetadata;
             Access = Internal;
+            ObsoleteReason = 'This field is replaced by field 16 - "Temp. Current Sequence No.".';
+#pragma warning disable AS0115
+            ObsoleteState = Removed;
```

Comment:
Is it better to be first pending and then with pragma also Removed in another version?

Reply:
I believe this should be fine, it's an internal field that should not be used by anyone else.

Reply:
This is no longer relevant as we now have capability of upgrading integer field to BigInteger.